holoviews/plotting/bokeh/element.py

```diff
@@ -168,18 +166,56 @@ def __init__(self, element, plot=None, **params):
         self.current_ranges = None
         super(ElementPlot, self).__init__(element, **params)
         self.handles = {} if plot is None else self.handles['plot']
-        element_ids = self.hmap.traverse(lambda x: id(x), [Element])
```

Comment:
I realized that even if a HoloMap of Overlays has only a single element it's still static.